chromium_src/net/url_request/redirect_util.cc

```diff
@@ -26,12 +29,15 @@ void RedirectUtil::UpdateHttpRequest(
                                  modified_headers,
                                  request_headers,
                                  should_clear_upload);
-  // Hack for dropping referrer at the network layer.
+  // Hack for capping referrers at the network layer.
   if (removed_headers) {
-    if (removed_headers->end() != std::find(removed_headers->begin(),
-                                            removed_headers->end(),
-                                            "X-Brave-Clear-Referer")) {
-      const_cast<RedirectInfo&>(redirect_info).new_referrer.clear();
+    if (base::Contains(*removed_headers, "X-Brave-Cap-Referer")) {
```

Comment:
Nit: `X-Brave-Cap-Referer` -> Refer**R**er? (if so, also in `browser/net/brave_site_hacks_network_delegate_helper.cc`)